drika_hotspot/Data/Scripts/Hotspots/drika_hotspot.as

```diff
@@ -516,7 +516,6 @@ void DrawEditor(){
 		ImGui_SetNextWindowSize(vec2(600.0f, 400.0f), ImGuiSetCond_FirstUseEver);
 		ImGui_SetNextWindowPos(vec2(100.0f, 100.0f), ImGuiSetCond_FirstUseEver);
 		ImGui_Begin("Drika Hotspot " + (show_name?" - " + display_name:"" + this_hotspot.GetID()) + "###Drika Hotspot", show_editor, ImGuiWindowFlags_MenuBar);
-		ImGui_PopStyleVar();

```

Comment:
At line 535 is also a ``ImGui_PopStyleVar();`` does that need to be removed as well?